core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/SequencerPartitionContainer.java

```diff
@@ -46,30 +46,33 @@
 @JsonSerialize(typing = JsonSerialize.Typing.STATIC, include = JsonSerialize.Inclusion.NON_NULL)
 @JsonTypeInfo(use = JsonTypeInfo.Id.CLASS, include = JsonTypeInfo.As.PROPERTY, property = "@class")
 @JsonIgnoreProperties({ "securityProfile", "run" })
-public interface SequencerPartitionContainer<T extends Partition> extends SecurableByProfile, Deletable, Comparable, Barcodable, Locatable {
+public interface SequencerPartitionContainer<T extends Partition>
+    extends SecurableByProfile, Deletable, Comparable<SequencerPartitionContainer<?>>, Barcodable, Locatable
+{
+
   public void setId(long id);
 
   /**
-   * Sets the name of this Container object.
+   * Returns the runs of this Container object.
    * 
-   * @param name name.
+   * @return Collection<Run> run.
    */
-  public void setName(String name);
+  Collection<Run> getRuns();
 
   /**
-   * Returns the run of this Container object.
+   * Sets the runs of this Container object.
+   * 
+   * @param runs The runs of which this Container is a part.
    * 
-   * @return Run run.
    */
-  Run getRun();
+  void setRuns(Collection<Run> runs);
 
   /**
-   * Sets the run of this Container object.
-   * 
-   * @param run The run of which this Container is a part.
+   * Returns the last run of this Container object.
    * 
+   * @return Run run
    */
-  void setRun(Run run);
+  Run getLastRun();
```

Comment:
Please Javadoc this to note what exactly it's expected to return (The run with the latest start date of those which have a known status?)